test/UnitTest/TestingControllers/InstancesControllerTests.cs

```diff
@@ -622,6 +623,87 @@ public async Task GetEmptyListOfInstances_UserRequestsInstancesNoPartyIdDefinedA
             registerService.VerifyAll();
         }
 
+        /// <summary>
+        /// Test case: Get Multiple instances with wrong organisation number and without specifying instance owner partyId.
+        /// Expected: Register service throws 400 bad request.
+        /// </summary>
+        [Fact]
+        public async Task GetMany_UserRequestsInstancesNoPartyIdDefinedAndWithWrongOrganisation_ThrowsBadRequest()
+        {
```

Comment:
A bit confused by this test. orgno is 9 digits and should be valid based on the error message. So not sure what "wrong" org number means  here?

Reply:
Spot on! I agree this test is misleading because of incorrect message and uses of "wrong". 👍